### PR TITLE
[TUTORIAL] links to imprint and privacy policy changed to human-connection.org testvideo ;)

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
links to imprint and privacy policy changed to human-connection.org
as discussed with martin I exchanged the links in the footer.
the imprint and the privacy policy go to human-connection.org .. on the respective page.

links von impressum und datenschutz auf human-connection.org geändert
das PR ist auch ein Testvideo. 

is the same PR: https://github.com/Human-Connection/Human-Connection/pull/1615

_____

https://www.youtube.com/watch?v=EeVspZ8jC6I

Ablauf und Schritte Zeitangaben:

01:02 Hinweis auf Human Connection Discord Chat

04:14 - Starten der Lokalen Anwendung Human Connection

05:07 - Neo4j Datenbank füllen

05:30 - Problem der Aufgabe

06:32 - Coden

13:32 - Vorbereitung für git push

15:53 - Test Lint

18:03 - Test Jest

20:18 - Git Push

21:30 - Pull Request schreiben

24:10 - Travis Tests

27:24 - Abschluss

Das Video stellt den Zeitablauf stark verkürzt wieder. Wir haben einige langweilige stellen geschnitten. Der größte zeitliche aufwand, sind die Tests und das diese im grünen Bereich landen.

Bei Fragen kommt in den Discord Chat und sprecht mit den Entwicklern.

### CODE REVIEW

File: webapp/components/PageFooter/PageFooter.vue [hunk 1]
```diff
@@ -2,13 +2,21 @@
   <div id="footer" class="ds-footer">
     <a href="https://human-connection.org" target="_blank" v-html="$t('site.made')"></a>
     <span>-</span>
-    <nuxt-link to="/imprint">{{ $t('site.imprint') }}</nuxt-link>
+    <a
+      href="https://human-connection.org/impressum/"
+      target="_blank"
+      v-html="$t('site.imprint')"
+    ></a>
```
Comment (roschaefer) [inline webapp/components/PageFooter/PageFooter.vue]: What's the reason to remove the imprint, too? Keep in mind that at some point we have multiple instances of human connection. On that day, both data privacy section and imprint must be part of the code anyways. Pointing the data privacy to https://human-connection.org/en/privacy/ is a temporary workaround so it's easier for @datenbrei to keep it updated. Does the imprint also needs to be updated that often? Or can we just keep it?

Comment (ogerly) [reply webapp/components/PageFooter/PageFooter.vue]: if we create an area in the admin area where these files can be changed.

**Comment by roschaefer** [comment]
@ogerly can you have a look into this? Maybe you reverted your changes in the merge commit? Merging this PR would only change whitespaces!

**Comment by roschaefer** [comment]
This PR contains changes to whitespaces only, so I'm going to close this @ogerly 

I will not delete the branch though, in case you still need it.

**Pull request closed without merging**